### PR TITLE
[VEN-2102]: add PLANET market to DeFi Pool

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-2102

### CODE REVIEW

File: vips/vip-197/vip-197.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            ],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: "0x2b8A1C539ABaC89CbF7E2Bc6987A0A38A5e660D4",
+        signature: "upgradeTo(address)",
+        params: ["0x1Db646E1Ab05571AF99e47e8F909801e5C99d37B"],
+      },
```
Comment (chechu) [inline vips/vip-197/vip-197.ts:56]:
```suggestion
```

File: vips/vip-197/vip-197.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      {
+        target: VPLANET_DEFI,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [28800],
+      },
+      {
+        target: VPLANET_DEFI,
+        signature: "setProtocolShareReserve(address)",
+        params: [PSR],
+      },
```
Comment (chechu) [inline vips/vip-197/vip-197.ts:67]: I think we can avoid this. See [this](https://github.com/VenusProtocol/isolated-pools/pull/323#pullrequestreview-1711128394)

File: vips/vip-197/vip-197.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VPLANET_DEFI,
+            parseUnits("0.2", 18),
+            parseUnits("0.3", 18),
+            parseUnits("174983000", 18),
+            VTOKEN_RECEIVER,
+            parseUnits("500000000", 18),
```
Comment (chechu) [inline vips/vip-197/vip-197.ts:94]:
```suggestion
            parseUnits("1000000000", 18),
```

File: simulations/vip-197/vip-197/simulations.ts [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+      });
+
+      it("should register vPLANET_DeFi in PoolRegistry", async () => {
+        const vToken = await poolRegistry.getVTokenForAsset(COMPTROLLER, PLANET);
+        expect(vToken).to.equal(VPLANET_DEFI);
+      });
+    });
+
+    describe("Ownership", () => {
+      it("should transfer ownership of vUSDD_Stablecoins to Timelock", async () => {
```
Comment (chechu) [inline simulations/vip-197/vip-197/simulations.ts:75]:
```suggestion
      it("should transfer ownership of vPLANET_DeFi to Timelock", async () => {
```

File: simulations/vip-197/vip-197-testnet/simulations.ts [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+      });
+
+      it("should register vPLANET_DeFi in PoolRegistry", async () => {
+        const vToken = await poolRegistry.getVTokenForAsset(COMPTROLLER, PLANET);
+        expect(vToken).to.equal(VPLANET_DEFI);
+      });
+    });
+
+    describe("Ownership", () => {
+      it("should transfer ownership of vUSDD_Stablecoins to Timelock", async () => {
```
Comment (chechu) [inline simulations/vip-197/vip-197-testnet/simulations.ts:79]:
```suggestion
      it("should transfer ownership of vPLANET_DeFi to Timelock", async () => {
```

File: vips/vip-197/vip-197.ts (hunk 1 above)
Comment (GitGuru7) [reply vips/vip-197/vip-197.ts:56]: We need this command for success of mainnet simulation until this proposal get executed https://app.venus.io/#/governance/proposal/194

File: vips/vip-197/vip-197-testnet.ts [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      },
+      {
+        target: REWARD_DISTRIBUTOR,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: REWARD_DISTRIBUTOR,
+        signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+        params: [[VPLANET_DEFI], ["1860"], ["1860"]],
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-197/vip-197-testnet.ts:100]: USDT in testnet has 6 decimals. `1860` is correct

File: vips/vip-197/vip-197.ts (hunk 1 above)
Comment (chechu) [reply vips/vip-197/vip-197.ts:55]: Understood. Thanks!

**Review by chechu — COMMENTED** [review]
LGTM

File: vips/vip-197/vip-197.ts (hunk 1 above)
Comment (chechu) [reply vips/vip-197/vip-197.ts]: Removed https://github.com/VenusProtocol/vips/pull/102/commits/02779346bcf3743bd12a838d6d1ae4929d1986d5

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/198

**Pull request merged**